man/systemd-system.conf.xml

```diff
@@ -382,30 +382,35 @@
-        itself.</para></listitem>
+        <listitem><para>These settings control various default resource limits for units. See
+        <citerefentry><refentrytitle>setrlimit</refentrytitle><manvolnum>2</manvolnum></citerefentry> for
+        details. The resource limit is possible to specify in two formats, <option>value</option> to set soft
+        and hard limits to the same value, or <option>soft:hard</option> to set both limits individually
+        (e.g. DefaultLimitAS=4G:16G).  Use the string <varname>infinity</varname> to configure no limit on a
+        specific resource. The multiplicative suffixes K (=1024), M (=1024*1024) and so on for G, T, P and E
+        may be used for resource limits measured in bytes (e.g. <literal>DefaultLimitAS=16G</literal>). For
+        the limits referring to time values, the usual time units ms, s, min, h and so on may be used (see
+        <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry> for
+        details). Note that if no time unit is specified for <varname>DefaultLimitCPU=</varname> the default
+        unit of seconds is implied, while for <varname>DefaultLimitRTTIME=</varname> the default unit of
+        microseconds is implied. Also, note that the effective granularity of the limits might influence
+        their enforcement. For example, time limits specified for <varname>DefaultLimitCPU=</varname> will be
+        rounded up implicitly to multiples of 1s. These settings may be overridden in individual units using
+        the corresponding <varname>LimitXXX=</varname> directives. Note that these resource limits are only
+        defaults for units, they are not applied to the service manager process (i.e. PID 1) itself.</para>
+
+        <para>When these settings are configured for the per-user instance of the service manager (i.e. in
+        the user-modifiable <filename>user.conf</filename> rather than privileged
+        <filename>system.conf</filename> file) they cannot be used to raise the limits above those set when
+        the service manager is first invoked, as the user's service manager generally lacks the privileges to
+        do so. In user context these configuration options are hence only useful to lower the limits passed
+        in or to raise the soft limit to the maximum of the hard limit as configured for the user. To raise
+        the user's limits further, the available configuration mechanisms differ between operating systems,
+        but typically require privileges. In most cases it is possible to configure higher per-user resource
+        limits via PAM or by setting limits on the system service encapsulating the user service manager,
+        i.e. the user's instance of <filename>user@.service</filename>. After making such changes, make sure
+        to fully log out and re-login, ensuring the per-user service manager is terminated and restarted
+        inbetween.</para></listitem>
```

Comment:
No, this does not work, because of lingering and the manager shutdown delay too. It is necessary to restart the user manager.